api/project/v1/project.proto

```diff
@@ -22,9 +22,9 @@ service ProjectAPI {
 message GetProjectsRequest {
   repeated string projects = 1;
   repeated string users = 2;
-  // If this is set to true, we will compute all project dependencies for the requested projects.
+  // If this is set to true, the api will return all project dependencies for the requested projects.
   // By default this is set to true.
```

Comment:
it's not possible with the current api to determine whether it was set explicitly to false or not

Reply:
Oh yeah true since we dont implement in OSS, let me remove this.